code/__DEFINES/traits/declarations.dm

```diff
@@ -803,6 +803,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_THROWINGARM "throwing_arm"
 #define TRAIT_SETTLER "settler"
 #define TRAIT_STRONG_STOMACH "strong_stomach"
+#define TRAIT_MASQUERADE_FOOD "masquerade_food"
```

Comment:
Put this in the ~~bubber __DEFINES folder.

Reply:
Which one is that?

Reply:
NVM, found it I think!